src/main/java/uk/gov/hmcts/reform/sandl.snlapi/controllers/RootController.java

```diff
@@ -25,4 +30,19 @@
     public ResponseEntity<String> welcome() {
         return ok("Welcome to snl-api");
     }
+
+    @CrossOrigin
+    @RequestMapping(path = "/get-sessions", method = RequestMethod.GET, produces = "application/json")
+    @ResponseBody
+    public String getSessions(@RequestParam("date") String date) {
```

Comment:
Let's keep the RootController only for azure ping request which is  this welcome endpoint). Other requests methods  please put into relevant other/new controllers.